x11-wm/mutter/mutter-9999.ebuild

```diff
@@ -110,8 +110,9 @@ RDEPEND="
 	)
 "
 DEPEND="${RDEPEND}
-	x11-base/xorg-proto
 	sysprof? ( >=dev-util/sysprof-common-3.38.0 )
+	X? ( x11-base/xorg-proto )
+	xwayland? ( x11-base/xorg-proto )
```

Comment:
Did you try building it without xorg-proto without the USE flags? It can be quite hard for upstream to be clean on this and not have an accidental include somewhere.